tests/test_apiclient.py

```diff
@@ -400,6 +400,15 @@ def test_get_aggregations_url(self, search_client):
     def test_get_index_from_search_api_url(self, search_client, search_api_url, expected_index):
         assert search_client.get_index_from_search_api_url(search_api_url) == expected_index
 
+    def test_search_service_from_url(self, search_client, rmock):
+        rmock.get(
+            'http://baseurl/g-cloud/services/search?lot=cloud-hosting',
+            json={'services': "myresponse"},
+            status_code=200)
+        result = search_client.search_services_from_url(search_api_url='https://baseurl/g-cloud/services/search'
```

Comment:
This might be a personal thing, but I find it's usually better to avoid the implicit string concatenation. Most of the time it looks like there's a missing comma between two arguments. At least that's how I'd read this code after a quick glance.

Something like this feels less ambiguous:

```
result = search_client.search_services_from_url(
    search_api_url='https://baseurl/g-cloud/services/search?lot=cloud-hosting'
)
```

Reply:
Reasonable. I'll change this.